pkg/providers/azure.go

```diff
@@ -164,8 +164,9 @@ func (sc *SetUpCmd) CreateServicePrincipal() error {
 
 func (sc *SetUpCmd) assignSpRole() error {
 	log.Debug("Assigning contributor role to service principal...")
+	contributorId := "b24988ac-6180-42a0-ab88-20f7382dd24c"
```

Comment:
💡 Nice, just an idea - I think this `guid` refer to this doc - https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#all might as well comment above the implementation for readers who might wonder what this guid translation leads to.